config-overrides.js

```diff
@@ -0,0 +1,4 @@
+const { override, addBabelPreset } = require('customize-cra')
+module.exports = override(
+  addBabelPreset('@emotion/babel-preset-css-prop')
```

Comment:
Latest version of @mui works with emotion. We need to override CRA's default config